frontend/src/toolbar/experiments/ExperimentsEditingToolbarMenu.tsx

```diff
@@ -0,0 +1,208 @@
+import { IconPlus, IconSearch, IconTrash } from '@posthog/icons'
+import { LemonDivider } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { Form, Group } from 'kea-forms'
+import { EditableField } from 'lib/components/EditableField/EditableField'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+
+import { ToolbarMenu } from '~/toolbar/bar/ToolbarMenu'
+import { experimentsTabLogic } from '~/toolbar/experiments/experimentsTabLogic'
+import { WebExperimentTransformField } from '~/toolbar/experiments/WebExperimentTransformField'
+
+export const ExperimentsEditingToolbarMenu = (): JSX.Element => {
+    const { selectedExperimentId, inspectingElement, experimentForm } = useValues(experimentsTabLogic)
+    const {
+        setExperimentFormValue,
+        selectExperiment,
+        selectVariant,
+        inspectForElementWithIndex,
+        deleteExperiment,
+        newVariant,
+        newElement,
```

Comment:
I'd prefer more descriptive names for these actions: `addNewVariant`, `addNewElement`